src/tool/cpp/cppwinrt/component_writers.h

```diff
@@ -138,7 +138,7 @@ namespace xlang
     {
         w.write_root_include("base");
         auto format = R"(%
-bool WINRT_CALL %_can_unload_now() noexcept
+bool WINRT_CALL winrt_can_unload_now() noexcept
```

Comment:
No, this is wrong. It's very important that this preamble to these functions is still configurable.